src/accounts/decoders.rs

```diff
@@ -38,6 +38,17 @@ pub(crate) fn position(message: &mut ResponseMessage) -> Result<Position, Error>
     Ok(position)
 }
 
+pub(crate) fn family_code(message: &mut ResponseMessage) -> Result<FamilyCode, Error> {
```

Comment:
This should return a vector similar to the contract_descriptions message https://github.com/wboayue/rust-ibapi/blob/723634d2d03bf91fc15c57c7610cd7cb6938529b/src/contracts/decoders.rs#L127.

This is a single message. Which contains a count for the number of family codes.

Reply:
Will do.  Thanks